### PR TITLE
added display for test metrics in predict model

### PR DESCRIPTION
**Pull request by ngupta23**
#### Describe the changes you've made
* predict_model displays the test metrics now when verbose is set to True (default)
* Note that this is slightly different from the regression.classification case. There are several corner cases such as: 
  - If the model has not been finalized, prediction match y_test
  - If the model has been finalized, predictions will not match the indices from y_test (metrics display NaN in that case)
  - Also, the user can use a different fh length in predict compared to what was used in setup. In this case, only the common indices in prediction and test are used to compute the metrics.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Unit test has been added

#### Describe if there is any unusual behaviour of your code

Warnings are issued for all corner cases identified above

## Checklist:
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by Yard1** [comment]
@ngupta23 Thanks, I'll edit `create_model` and we can merge this

**Comment by ngupta23** [comment]
> 
> 
> @ngupta23 Thanks, I'll edit `create_model` and we can merge this

@Yard1  What is the change needed to `create_model`?

**Comment by Yard1** [comment]
@ngupta23 Displaying the frame returned by `predict_model` if `cross_validation=False`

File: pycaret/internal/pycaret_experiment/time_series_experiment.py [hunk 1]
```diff
@@ -2024,9 +2024,30 @@ def predict_model(
...
 
+        display = None
+        try:
+            np.random.seed(self.seed)
+            if not display:
+                display = Display(
+                    verbose=verbose,
+                    html_param=self.html_param,
+                )
+        except:
```
Comment (TremaMiguel) [inline pycaret/internal/pycaret_experiment/time_series_experiment.py:2045]: can we catch here the expected error instead of catching all kind of errors ?

**Review by TremaMiguel — APPROVED** [review]

File: pycaret/internal/pycaret_experiment/time_series_experiment.py (hunk 1 above)
Comment (ngupta23) [reply pycaret/internal/pycaret_experiment/time_series_experiment.py:2045]: I have not delved into this part. This is copied over from the regression module. @Yard1 any thoughts on this?

Comment (Yard1) [reply pycaret/internal/pycaret_experiment/time_series_experiment.py:2045]: It would be a good idea, but I can't remember what sort of an error can pop up there
